lib/parser/message.h

```diff
@@ -207,6 +208,7 @@ class Messages {
 
   void Incorporate(Messages &);
   void Copy(const Messages &);
+  void ResolveProvenances(const CookedSource &);
```

Comment:
A comment about what this does would be helpful. As I understand it, it converts message locations to ProvenanceRange from CharBlock when it can.

Reply:
adding one